bills-app/src/main/webapp/script.js

```diff
@@ -767,26 +767,55 @@ const mockResponse = {
     }]
 }
 
-// Fetched the upcoming bills from the api
-const getUpcomingBills = () => {
+function getUpcomingBills() {
     //api key is declared in api.js as: 'proPublicaApiKey'
+    // clearBillData();
     const url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
     fetch(url, {
         method: "GET", 
         headers: {
             "X-API-Key": proPublicaApiKey
         }
     }).then(response => response.json()).then((upcomingBills) => {
-        //printing the json file to console
-        console.log(upcomingBills);
+        //Prints out the contents of "status" from house.json
+        //If everything goes well, prints out "OK" to console
+        console.log(upcomingBills.status);
+        clearBillData();
 
-        //Prints out the contents of "status" from house.json into the website. 
-        //If everything goes well, prints out "OK"
-        document.getElementById('API-status-container').innerText = upcomingBills.status;
+        //the api call to getting upcoming bills is not returning any bills.
+        //this renders all upcoming bills to the main page 
+        // upcomingBills.results[0].bills.map(bill => {
+        //     renderBillFromID(bill.congress, bill.bill_slug);
+        // });
```

Comment:
You can leave the real code for getUpcomingBills that handles the actual (now empty) response and delete mockResponse